core/src/main/java/nl/nn/adapterframework/webcontrol/api/ShowLiquibaseScript.java

```diff
@@ -79,7 +79,7 @@ public Response downloadScript() throws ApiException {
 
 		for(Configuration config : getIbisManager().getConfigurations()) {
 			DatabaseMigratorBase databaseMigrator = config.getBean("jdbcMigrator", DatabaseMigratorBase.class);
-			if(databaseMigrator.isEnabled()) {
```

Comment:
I would personally prefer a big fat warning that somebody enabled the migrator but didn't supply the migration script.
I think the real issue here, is that liquibase can be enabled from the 'src/main/resources' AppConstants instance. It should only check in it's own context (just like the migration script.

Reply:
It is not necessary to set a property to use this tool. Having a script in the configuration classpath is sufficient.